ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__azure-4.17-nightly-x86.yaml

```diff
@@ -159,6 +159,30 @@ tests:
     - chain: openshift-qe-control-plane
     workflow: openshift-qe-installer-azure
   timeout: 5h0m0s
+- always_run: false
+  as: node-density-heavy-24nodes
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: azure-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      ADDITIONAL_WORKER_ARCHITECTURE: aarch64
+      ADDITIONAL_WORKER_VM_TYPE: Standard_D4ps_v5
+      ADDITIONAL_WORKERS: "3"
+      COMPUTE_NODE_REPLICAS: "3"
+      COMPUTE_NODE_TYPE: Standard_D4s_v5
+      CONTROL_PLANE_INSTANCE_TYPE: Standard_D4s_v5
+      CUSTOM_AZURE_REGION: eastus
+      OCP_ARCH: amd64
```

Comment:
You will also need to add `      ITERATIONS_PER_NAMESPACE: "400"`, as its needed to run node-density-heavy jib

Reply:
@shahsahil264 Do you remember why 400 is needed? I searched our configs, we only have ITERATIONS_PER_NAMESPACE: "400" in Nutanix jobs, firstly added by Liquan in https://github.com/openshift/release/pull/54337. 
Otherwise the default in ref is [1000](https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift-qe/node-density-heavy/openshift-qe-node-density-heavy-ref.yaml#L37.) which should be used in other node-density-heavy jobs.

Reply:
+1 (wondering the same)

Reply:
@qiliRedHat I don't remember exact reason, I maybe based my previous jenkins job testing setting, it should no special reason, we can use the same setting with other cloud providers.

Reply:
@liqcui Thanks for the clarification. Then let's remove it on Nutanix jobs to see if the default can work. It's not a hurry.